components/application-operator/charts/application/values.yaml

```diff
@@ -50,3 +50,11 @@ eventService:
   service:
     externalapi:
       port: *eventServiceExternalAPIPort
+
+# NOTE(antoineco): the "sources.kyma-project.io" API group is registered by the
+# "event-sources" component, which is currently part of the optional (alpha)
+# Knative Event Mesh. The HTTP event source is therefore puposedly ignored
+# until that component becomes part of the default Kyma installation.
+# Ref. kyma-project/kyma#5927
+knativeEventMesh:
```

Comment:
This is insufficient to toggle off the complete feature. Validator will still expose `/{application}/events` endpoint.

Reply:
That's OK, the first thing we'll do after merging is enable the event-sources by default and remove those lines. We _do_ want the validator to have this endpoint exposed ASAP.